### PR TITLE
Rewrite feature list

### PR DESCRIPTION
**Pull request by sudormrfbin**
Not tested on mobile.

Screenshot_2022-03-12_11-14-03

### CODE REVIEW

**Review by pickfire — COMMENTED** [review]
Looks quite bad on mobile after I tested it with responsive view in browser.

image

File: static/style.css [hunk 1]
```diff
@@ -95,3 +95,20 @@ pre {
...
 }
+
+.selling-points {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.selling-point {
+  /* 3 selling points per row */
+  width: 26%;
```
Comment (pickfire) [inline static/style.css:106]:
```suggestion
  /* 2 selling points per row */
  max-width: 20em;
```
I think having two selling points will make it look better and setting the max width.

image

image

File: templates/index.html [hunk 2]
```diff
@@ -5,45 +5,71 @@ <h2 class="subtitle">A post-modern <strong>text editor</strong>.</h2>
...
+
           <h3>Post-modern?!</h3>
-          <p>It's a joke. If neovim is the modern vim, then helix is post-modern.</p>
+          <p>It's a joke. If Neovim is the modern Vim, then Helix is post-modern.</p>
+
           <h3>Is it any good?</h3>
           <a href="http://news.ycombinator.com/item?id=3067434">Yes.</a>
+
+          <h3>Are there plans for a GUI frontend?</h3>
+          <p>Eventually, yes ! See the <a href="https://github.com/helix-editor/helix/issues/39">
```
Comment (pickfire) [inline templates/index.html:70]:
```suggestion
          <p>Eventually, yes! See the <a href="https://github.com/helix-editor/helix/issues/39">
```

File: static/style.css (hunk 1 above)
Comment (sudormrfbin) [reply static/style.css]: Thanks for the suggestion, looks much better 👍🏾

**Comment by sudormrfbin** [comment]
Changed to two features per row normally and 1 feature per row on mobile:

| Browser | Mobile |
| --- | --- |
| helix site normal | helix site mobile |

File: templates/index.html [hunk 3]
```diff
@@ -5,45 +5,71 @@ <h2 class="subtitle">A post-modern <strong>text editor</strong>.</h2>
...
-          <p>The base implementation uses a terminal frontend and combines well with tmux or usage over ssh.</p>
-          <p>I'm also exploring alternate frontends (wgpu-rs, skia/skulpin based
-          rendering).</p>
-
-          <h2>Built with <strong>Rust</strong>. No Electron. No VimScript. No JavaScript.</h2>
-          <p>Your laptop battery life will thank you.</p>
+        <div class="selling-points">
+          <div class="selling-point">
+            <h2>Multiple selections</h2>
+            Multiple cursors as a core editing primitive, inspired from
```
Comment (the-mikedavis) [inline templates/index.html:15]:
```suggestion
            Multiple cursors as a core editing primitive, inspired by
```

I think this reads better but I would be ok with `from` as well

**Review by the-mikedavis — APPROVED** [review]
tested out locally and looks great!

**Review by pickfire — APPROVED** [review]
Didn't test again but I think should be good.

**Pull request merged**